extensions/interactions/interaction_specs.json

```diff
@@ -200,8 +200,8 @@
     }
   },
   "ItemSelectionInput": {
-    "can_have_solution": false,
-    "show_generic_submit_button": false,
+    "can_have_solution": true,
+    "show_generic_submit_button": true,
```

Comment:
What does this change do?

Reply:
setting `show_generic_submit_button` to true allows a submit button to be displayed when adding a solution.
(This frame from the first video):
![image](https://user-images.githubusercontent.com/78612244/158194489-1acbc200-401e-415e-bc2e-96a068b5e362.png)

This button is required for the solution choices to be saved, only after which the explanation text box is made available to the creator (the same can be seen in the first video).